docs/source/model_cards/cifar_resnet.md

```diff
@@ -39,15 +40,33 @@ The biggest differences between CIFAR ResNet models and ImageNet ResNet models a
 | ResNet44             | 0.66M           | TBA          | TBA                     |
 | ResNet56             | 0.85M           | 93.1%        | 35 min                  |
 | ResNet110            | 1.7M            | TBA          | TBA                     |
-
 ## Default Training Hyperparameters
 
-- Optimizer: SGD
-    - Learning rate: 1.2
-    - Momentum: 0.9
-    - Weight decay: 1e-4
-- Batch size: 1024
-- LR Schedulers
-    - Linear warmup for 5 epochs
-    - Multistep decay by 0.1 at epochs 80 and 120
-- Number of epochs: 160
\ No newline at end of file
+```yaml
+optimizer:
```

Comment:
Could we provide a functional code example that someone could run with these values? I.e. that does not depend on yahp?
Ideally the code is in a separate python file that has a test case to ensure that the trainer initializes (and maybe trains a batch on dummy data)

Reply:
Oh this was not supposed to be a YAHP-related example at all. I just used yaml format because it was the cleanest way to show a list of hyperparameters used for training (kind of like how NVIDIA DL Examples shows config files for all their models). 

The point of this example is just to show user's what values for each parameter they can use when training with the model (not necessarily how to use the model with our trainer) in a relatively clean UI. There are plenty of non-yahp examples in the upcoming new `Using the Trainer` section of the docs.